frontend/src/features/upload/components/CoverLetterContentArea.tsx

```
@@ -0,0 +1,38 @@
+            극복하셨나요?
+          </div>
+        </div>
+        <div className='flex flex-col gap-3 pl-13'>
+          <div className='text-sm text-gray-400'>총 1,038자</div>
+          <div className='text-sm text-gray-600'>대학 시절 ...</div>
+        </div>
+      </div>
+      <div className='flex gap-5 justify-self-center'>
+        <button className='rounded-[0.438rem] bg-gray-50 p-2 cursor-pointer'>
+          <I.LeftPaginationButtonIcon color='#D9D9D9' />
+        </button>
```

Biome (2.3.13)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)